tests/transforms/test_experimental/test_transform_dispatcher.py

```diff
@@ -486,7 +486,10 @@ def comb_postproc(results: TensorLike, fn1: Callable, fn2: Callable):
 
         # Create a simple device and tape
         tmp_dev = qml.device("default.qubit", wires=3)
-        H = qml.PauliY(2) @ qml.PauliZ(1) + 0.5 * qml.PauliZ(2) + qml.PauliZ(1)
+
+        H = qml.Hamiltonian(
```

Comment:
Do you think for test cases like this where we are actually testing the Hamiltonian generated, we need to also test legacy opmath behaviour?

Reply:
We are already calling two different functions depending on whether opamth is enabled or disabled... I don't think so, but I am not sure

Reply:
Just to reiterate the point of `@pytest.mark.usefixtures("use_legacy_opmath")` is not to mark any test that is failing with new opmath and leave it with that. The main purpose of it is to test behavior that is explicitly not needed/supported on new opmath, e.g. everything in `tests/ops/qubit/test_hamiltonian.py`. Since Hamiltonians are used in other parts of the codebase, the fixture can be used there as well for ensuring backward compatibility.